test/e2e/storage/drivers/csi.go

```diff
@@ -310,8 +306,11 @@ func (g *gcePDCSIDriver) GetClaimSize() string {
 	return "5Gi"
 }
 
-func (g *gcePDCSIDriver) CreateDriver() {
+func (g *gcePDCSIDriver) CreateDriver(f *framework.Framework) (*testsuites.PerTestConfig, func()) {
+	framework.SkipIfMultizone(f.ClientSet)
```

Comment:
what do you think about passing framework into SkipUnsupportedTest instead?

Reply:
nvm, the point of this is to avoid initializing framework

Reply:
nvm = never mind? If so, then I'll ignore the question ;-}